lua/autorun/server/bounties.lua

```diff
@@ -23,39 +27,57 @@ end
 function AddBounty(ply,target,amount)
 	if !ply:IsPlayer() or !target:IsPlayer() or type(amount) != "number" then return end
 	amount = amount > 0 and amount or 0
-	if ply:PS_HasPoints(amount) then
-		SetPlayerBounty(target,GetPlayerBounty(target) + amount)
+	if BountyLimit[ply:SteamID()] == nil then BountyLimit[ply:SteamID()] = 0 end
+	local total = BountyLimit[ply:SteamID()] + amount
+	if ply:PS_HasPoints(amount) and total <= 10000000 then
+		local add = GetPlayerBounty(target) + amount
+		BountyLimit[ply:SteamID()] = BountyLimit[ply:SteamID()] + amount
+		SetPlayerBounty(target,add)
 		ply:PS_TakePoints(amount)
-		BotSayGlobal("[fbc]"..target:Nick().."'s bounty is now [rainbow]"..GetPlayerBounty(target) + amount.." [fbc]points")
+		BotSayGlobal("[fbc]"..target:Nick().."'s bounty is now [rainbow]"..add.." [fbc]points")
+	elseif ply:PS_HasPoints(amount) and total > 10000000 then
+		ply:ChatPrint("[red]You have reached your limit for today")
 	else
 		ply:ChatPrint("[red]You don't have enough points")
 	end
 end
 
 RegisterChatCommand({'bounty','setbounty'},function(ply,arg)
 	local t = string.Explode(" ",arg)
-	local to,c = PlyCount(t[1])
+	local to,c = PlyCount(string.Implode(table.remove(t)))
 	if to:IsPlayer() and c == 1 then
 		local bounty = GetPlayerBounty(to)
-		local p = tonumber(t[2])
+		local p = tonumber(t[#t])
 		if p != nil then
 			if p >= 1000 then
 				AddBounty(ply,to,p)
 			else
 				ply:ChatPrint("[red]You must add a minimum of 1000 points to the bounty")
 			end
-		elseif bounty > 0 then
-			BotSayGlobal(to:Nick().."'s bounty is [rainbow]"..bounty.." [fbc]points")
-		elseif bounty == 0 then
-			BotSayGlobal(to:Nick().." has no bounty")
 		end
 	elseif c>1 then
-		ply:ChatPrint("[fbc]More than one person found with that string in their name")
+		ply:ChatPrint("[red]More than one person found with that string in their name")
 	else
-		ply:ChatPrint("[fbc]!bounty player points")
+		ply:ChatPrint("[orange]!bounty player points")
 	end
 end,{global=true,throttle=true})
 
+RegisterChatCommand({'showbounty'},function(ply,arg)
+	local to,c = PlyCount(arg)
+	if to:IsPlayer() and c == 1 then
+		local bounty = GetPlayerBounty(to)
+		if bounty > 0 then
+			ply:ChatPrint("[orange]"..to:Nick().."'s bounty is [edgy]"..bounty.." [orange]points")
+		else
+			ply:ChatPrint("[orange]"..to:Nick().." has no bounty")
+		end
+	elseif c>1 then
+		ply:ChatPrint("[red]More than one person found with that string in their name")
```

Comment:
you probably don't need to check this here since it's just informational

Reply:
I don't understand what you want me to do with it

Reply:
I don't know why I would remove that check since I only want that message appearing under that circumstance and I have an else for a different message

Reply:
I mean, if there are multiple  people which match you can just post any one of them. the plycount thing is intended to ensure you don't accidentally send points to the wrong person but it's not necessary to be as certain when you're just printing information